app/pods/publications/controller.js

```diff
@@ -33,6 +33,13 @@ export default class PublicationsController extends Controller {
     yield this.search(this.searchText);
   }
 
+  constructor() {
+    super(...arguments);
+    this.publicationService.getNewPublicationNextNumber(false).then((res) => {
+      this.publication.number = res;
+    });
```

Comment:
Je gaat het nieuwe nummer niet willen laten genereren in de constructor maar op het moment dat je effectief de generatie gaat doen (klikken op de knop publicatie aanmaken)